go.mod

```diff
@@ -6,141 +6,139 @@ replace github.com/urfave/cli => github.com/keep-network/cli v1.20.0
 
```

Comment:
Do not merge now. There are some occasional test failures (only remotely) that need to be investigated, they may be due to the changes introduced by this PR.